tdrs-backend/tdpservice/settings/common.py

```diff
@@ -361,3 +363,14 @@ class Common(Configuration):
     )
 
     ENABLE_DEVELOPER_GROUP = True
+
+    # AMS OpenID #
+    AMS_CONFIGURATION_ENDPOINT = os.getenv(
+        'AMS_CONFIGURATION_ENDPOINT',
+        'https://shspfm-dev.gss.acf.hhs.gov/eaasidentityserver/.well-known/openid-configuration'
```

Comment:
Not sure if this should be kept here or reserved to .env vars, as this diverges from login.gov and its default values above.